frontend/ui/widget/configdialog.lua

```diff
@@ -1153,6 +1153,7 @@ function ConfigDialog:onConfigMoreChoose(values, name, event, args, name_text, m
                     curr_values = self.configurable[name]
                 end
                 widget = DoubleSpinWidget:new{
+                    width = not more_options_param.name and math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * 0.6) or nil,
                     title_text =  name_text or _("Set values"),
                     info_text = more_options_param.info_text,
```

Comment:
That's a bit cryptic :) What are the other config widgets using DoubleSpinWidget ? and are we sure they have or not a `more_options_param.name` ? And if they have one or not, we'll always be fine with 0.6 or the default ?
I was rather thinking about having an explicite setting in frontend/ui/data/creoptions.lua, among these:
https://github.com/koreader/koreader/blob/a21b5be64a25f33d3d8194b8b1282805f9a990c2/frontend/ui/data/creoptions.lua#L449-L465
something added like:
```lua
                             widget_width = 0.6,
```

Reply:
> and are we sure they have or not a `more_options_param.name` ?

Of course, (`left_min` and `name`) is unique for Word spacing.

I tried to avoid setting the width for each dialog if possible.

Reply:
> Of course, (left_min and name) is unique for Word spacing.

This works today, but think about the future. Safer and cleaner to be explicite.
Just require another `more_option_params` for those that wqnt a width different from the default of 0.8 for DoubleSpinWidget and 0.6 for SpinWidget.

`width = more_options_param.widget_width and math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * more_options_param.widget_width ) or nil,`

and add width=0.6 to the more_option_params of the 3 *_page_margins

You could even juste pass `width = more_options_param.widget_width` as 0.6 - and have SpinWidget compute a real width from the screen size if width > 0 and width < 1.

Reply:
> You could even juste pass `width = more_options_param.widget_width` as 0.6 - and have SpinWidget compute a real width from the screen size if width > 0 and width < 1.

Interesting, I think of it, may allow to remove a lot of code everywhere.

Reply:
Or a `width_ratio = 0.6` , if we don't want to mix real widths and ratio to smaller screen dimension.